pyflp/plugin.py

```diff
@@ -450,6 +472,80 @@ class FruityBalance(_PluginBase[FruityBalanceEvent], _IPlugin, ModelReprMixin):
     """
 
 
+class FruityBloodOverdrive(
+    _PluginBase[FruityBloodOverdriveEvent], _IPlugin, ModelReprMixin
+):
+    """![]()"""  # noqa
+
+    INTERNAL_NAME = "Fruity Blood Overdrive"
+
+    plugin_marker = _PluginDataProp[int]()  # 3
```

Comment:
No need to expose an internal parameter in the public interface.

I think this can be better exposed via a normal boolean `is_old` property.

Reply:
This was my workaround to get it to work with `StructEventBase`.
The proper way would probably be to read the first 4 bytes `self._stream.read_I()` and then set `is_old` in `NativePluginEvent`?

Reply:
Or maybe, it can be added as a property in `_PluginBase` itself